xbmc/cores/VideoPlayer/VideoPlayer.h

```diff
@@ -514,15 +515,17 @@ class CVideoPlayer : public IPlayer, public CThread, public IVideoPlayer,
       iSelectedSPUStream   = -1;
       iSelectedAudioStream = -1;
       iSelectedVideoStream = -1;
-      iDVDStillTime        =  0;
-      iDVDStillStartTime   =  0;
+      iDVDStillTime = std::chrono::milliseconds::zero();
```

Comment:
You use `std::chrono::milliseconds::zero()` here but `0ms` elsewhere?

Reply:
That's a good question. I believe the answer is that I don't want to include the `using namespace std::chrono_literals` in the header file.

Reply:
fair